### ISSUE TITLE
The sound of mobs can be heard from too far away

### ISSUE THREAD

**Issue by leorockway** [maikerumine/mobs_mc#109]
The sound of mobs can be heard from too far away which means that if you're in your house and there are mobs some 30 blocks away you can still hear them swimming, or walking on dirt or walking on snow and if there are many mobs it gets pretty overwhelming. I don't think that's how it works on MC.

**Comment by Wuzzy2** [comment]
I thought I set `distance=16` for all mobs, with exceptions for the bosses.

Are you only talking about footstep sounds? In that case, this is caused by the node definition, not the mob definition.

**Comment by leorockway** [comment]
Yes, I'm talking about footstep sounds / swimming sounds. I didn't know that was caused by the node definition. It is pretty annoying though. Particularly if you live near snow, the sound of a dozen animals walking on snow is a bit crazy.

**Comment by Wuzzy2** [comment]
Well, then there is not much we can do about in in `mobs_mc`. Talk to TenPlus1 in Mobs Redo.

**Comment by Wuzzy2** [comment]
Is this still an issue, or is this fixed now?

If not, please refer to the mods/game version you use (or commit ID, if on bleeding edge).

**Comment by tenplus1** [comment]
sounds.distance defaults to 10 in the api and can be made smaller, the footstep sounds on the other hand are handled by minetest and there's an issue open to fix: https://github.com/minetest/minetest/issues/4361
